content/en/funding/community-projects/index.mdx

```diff
@@ -1,97 +1,61 @@
 ---
-description: If you’ve got an idea for improving the MakerDAO community coordination or resources, we can provide up to 5,000 Dai to help make it happen.
+title: Community Projects
+keywords: Autonomous, Single Owner, Projects
 ---
 
-# Community Projects
+# Individual Contributions and Projects
 
-Got an idea for improving the MakerDAO community? Community Development can provide up to 5,000 Dai to help make it happen. In order to fund projects we have a useful program called  CDIPs.
+Got an idea and the gumption to improve the MakerDAO community? Community Development can provide up to 5,000 Dai to help make it happen. In order to fund self-directed projects we have a useful framework called Community Development Initiative Proposal (CDIP).
 
-The Community Development Initiative Proposal (CDIP) is the scoping and framing tool for anyone interested in receiving funding for proposing new MakerDAO community initiatives or resources. 
+A CDIP is the scoping and framing tool for anyone interested in receiving funds for proposing new MakerDAO community initiatives or resources.
 
 <Callout icon="question" secondary>
 
-If you are looking to contribute to an existing project with a defined scope, head over to our <Link to="/work-with-us/bounties">bounties page</Link>
+If you are looking to contribute to an existing project with a pre-defined scope, head over to our <Link to="/work-with-us/bounties/index.mdx">bounties page</Link>
 
 </Callout>
 
-## Propose Your Idea
-
-A proposal should combine suggested ideas (and skills) with the shared goal of improving the resources for the MakerDAO community. Follow these steps to start your proposal:
 
 <List>
 
+## _Examples_
 
 <Box>
 
-### 1. Write your proposal using our template.
-
-Use this Google Doc template to help structure a proposal so that information is clear and milestones are defined.
+Community Development outgrew the usecase of gitbooks and it created a plan to migrate [in 2019](https://github.com/makerdao/community/issues/225). Ryan Cordell executed a robust content revamp looking at the portal in an entirely new perspective.
 
-<Link to="https://docs.google.com/document/d/1lF8mNuomrguIS3lnvqTomS4NdON40nuyKRbLS6ZkMoA/edit">Open proposal template</Link>
+- [Content audit proposal](https://github.com/makerdao/community/issues/543)
 
 </Box>
 
-
 <Box>
 
-### 2. Post your proposal in our chat.
-
-Post your proposal in our <Link to="https://chat.makerdao.com/channel/community-development">Community-Development chat</Link>. An Advisor will reach out to help you through next steps.
-
-An Approving Group meets weekly to review proposals and will be in touch for regular updates. For more detail, read about the [community project process](./community-projects-process.md).
-
-</Box>
-
-
-</List>
-
-<CTA icon="question_mark" secondary>
-
-If you are unsure about your proposal, post your idea in the chat for feedback. An Advisor will help you iterate on your proposal.
-
-</CTA>
-
-<Accordion>
```

Comment:
Would recommend this as 2-column comparison since these are important to read right away.